pt-BR/active_record_validations.md

```diff
@@ -907,12 +908,12 @@ person.errors.messages
  # => {:email=>["has already been taken"], :age=>["is not a number"]}
 ```
 
-`person.valid?(:account_setup)` executes both the validations without saving
-the model. `person.save(context: :account_setup)` validates `person` in the
-`account_setup` context before saving.
+`person.valid?(:account_setup)` executa ambas as validações sem salvar
+o `model`. `person.save(context: :account_setup)` valida `person` no
```

Comment:
```suggestion
o *model*. `person.save(context: :account_setup)` valida `person` no
```

Parecido com alguns casos acima, é comum marcar essas palavras em itálico aqui no guia. Você pode ver também algumas palavras como essa no [Manual de tradução](https://github.com/campuscode/rails-guides-pt-BR/blob/master/TRANSLATION_MANUAL.md)